### PR TITLE
Update Week 1 course structure

### LINKED ISSUE #341: Detail the learning outcomes for the week
The learning outcomes are really broad and need to be more detailed as it will give the students a more precise overview of what exactly they'll be learning

### LINKED ISSUE #343: Update project description to clarify workflow
Since the [git workshop](https://github.com/foundersandcoders/git-workflow-workshop-for-two) is done in pairs, we will have to let the students know that since the actual projects are done in teams of 4, that they will be merging the other pair's pull request. This should be done in the project description of the week.

### LINKED ISSUE #344: Teach the students about the command line in a talk and/or point them to a resource
Since the command-line research topic isn't in the curriculum anymore in preference for a 'how to research afternoon' See #307. We would still like to familiarize them about the command-line/terminal and this can be done by either pointing them to a centralized resource and moving/updating this readme (https://github.com/FAC10/research/blob/master/week-1/customizing-command-line.md). It could also be done by @antoniotrkdz giving a talk.

### LINKED ISSUE #374: Replace git research topic with a topic on regex
Since most of the things asked in the Git research topic has already been covered in the [git workflow for two workshop](https://github.com/foundersandcoders/git-workflow-workshop-for-two) it's better to introduce a topic on regex's since that's something that is not covered in any of the weeks.

### PR DESCRIPTION
**Pull request by Samatar26**
Changed the research topics readme.
Fixes #374, #307 no longer viable for FAC11, but still open for future cohort(s) to work on. 
Added the command line resource to the project directory in week1
Fixes #344
Updated project description to reflect the changed git workshop
Fixes #343
Updated learning outcomes to be more detailed
Fixes #341

### CODE REVIEW

**Comment by jsms90** [comment]
Super speedy work @Samatar26. Thank you so much :tada: I will review this tomorrow, you absolute ninja.

One small thing - I don't think 304 was the issue that you meant to reference.

Also, try to keep your PRs granular. If we have no problems with your solution to 3 out of 4 issues, but we request extensive changes to the 4th issue, then those requests will hold up some perfectly merge-able work.

Seriously though, excellent work on getting to this in such a short space of time :heart_eyes_cat:

**Comment by Samatar26** [comment]
@eliascodes The command-line stuff is not the command-line workshop from week 4, but it's the research topic they were originally supposed to do in the previous cohorts [here](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/research-afternoon.md). It's purely been provided as extra-curricular reading. 

I understand everything else you've mentioned and will update, but don't understand what you mean by line 53 in project.md.

**Comment by eliascodes** [comment]
@Samatar26 

> Your team needs to set up a new repository under the relevant Github organisation - London: `FAC10` and Nazareth: `FAC1-Nazareth`

Remove these references to specific cohort repos. Instead say something like

> Your team needs to set up a new repository under your cohorts Github organisation.

**Comment by Samatar26** [comment]
@eliascodes can you review again, please?

File: coursebook/week-1/project.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ As a **client with JavaScript disabled in their browser for security reasons**
 
 ## Build Process (Wednesday-Thursday)
 
-Your team needs to set up a new repository under the relevant Github organisation - [London: `FAC10`](https://github.com/FAC10) and [Nazareth: `FAC1-Nazareth`](https://github.com/FAC1-Nazareth), and work together over the next few days to commit changes to this repository.
+Your team needs to set up a new repository under your cohorts Github organisation and work together over the next few days to commit changes to this repository.
```
Comment (jsms90) [inline coursebook/week-1/project.md]: @Samatar26 @antoniotrkdz still needs an update

**Comment by jsms90** [comment]
@Samatar26 @antoniotrkdz You need to resolve the merge conflicts please

@eliascodes could you re-review, given the extra addition to research afternoon #374

**Comment by jsms90** [comment]
@antoniotrkdz Please remove FAC10's resource on the command line

**Comment by Samatar26** [comment]
@jsms90 fixed merge conflicts

**Comment by jsms90** [comment]
@Samatar26 @antoniotrkdz Just waiting on a 2nd reviewer, now that you've made some more changes :+1:

File: coursebook/week-1/research-afternoon.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+
+## Regular Expressions
+1. Research:
+  + How to use regex
+    + (basic) pattern creation and repetitions
+    + (advanced) groups
+  + Find and use on-line tools such as [regex101](https://regex101.com/)
+2. Create a code snippet using regex such as (for example) email validator:
++ us3r1@foundersandcoders.com
+  + user must start with an alphabetical character;
```
Comment (eliascodes) [inline coursebook/week-1/research-afternoon.md]: [The correct terms](https://en.wikipedia.org/wiki/Email_address#Syntax) for the different parts of the e-mail address are "local-part" for what is referred to here as "user", and the "domain" for what is referred to here as the "site".

**Pull request merged**